src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/ParameterNameCheck.java

```diff
@@ -72,6 +78,12 @@
      */
     private boolean ignoreOverridden;
 
+    /** The visibility scope where methods are checked. */
+    private Scope scope = Scope.ANONINNER;
```

Comment:
@agcuda After talking with romani, we decided to leave it as `ANONINNER` so we can get this PR finished sooner. I will add this as a note to the referenced issue so it can be fixed there when we remove the enum.